calico/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -0,0 +1,164 @@
+---
+description: Understand how tiered policy works and supports microsegmentation.
+---
+
+# Get started with policy tiers
+
+## Seamless network policy integration
+
+**Network policy** is the primary tool for securing a Kubernetes network. It lets you restrict network traffic in your cluster so only the traffic that you want to flow is allowed. {{prodname}} provides more robust policy than Kubernetes, but you can use them together -- seamlessly. {{prodname}} supports:
+
+- {{prodname}} network policy, (namespaced)
+- {{prodname}} global network policy (non-namespaced, global)
+- Kubernetes network policy
+
+## Tiers: what and why?
+
+**Tiers** are a hierarchical construct used to group policies and enforce higher precedence policies that cannot be circumvented by other teams. As you will learn in this tutorial, tiers have built-in features that support workload microsegmentation.
+
+All {{prodname}} and Kubernetes network policies reside in tiers. You can start "thinking in tiers" by grouping your teams and types of policies within each group. For example, we recommend these three tiers (platform, security, and application).
+
+![policy-types](/img/calico-enterprise/policy-types.png)
+
+Next, you can determine the priority of policies in tiers (from top to bottom). In the following example, the platform and security tiers use {{prodname}} global network policies that apply to all pods, while developer teams can safely manage pods within namespaces using Kubernetes network policy for their applications and microservices.
+
+![policy-tiers](/img/calico-enterprise/policy-tiers.png)
```

Comment:
@ctauchen should we keep a copy the diagrams for Calico? Isn't it better to not keep multiple copies and use a shared one?